test/stacktraces.jl

```diff
@@ -0,0 +1,80 @@
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+# Tests for /base/stacktraces.jl
+
+# Some tests don't currently work for Appveyor 32-bit Windows
+const APPVEYOR_WIN32 = (
+    OS_NAME == :Windows && WORD_SIZE == 32 && get(ENV, "APPVEYOR", "False") == "True"
```

Comment:
If you can't figure this out, please do not merge this with the appveyor special-casing. If it fails on win32 and we don't know how to fix it yet, skip it all the time on win32, not just on appveyor. Tests that fail locally but not on CI is no good.